internal/chain/types.go

```diff
@@ -66,6 +66,7 @@ func (b *DagBlock) ToModel() (dag *models.Dag) {
 
 type Transaction struct {
 	models.Transaction
+	Logs	[]models.EventLog `json:"logs"`  
```

Comment:
Something wrong with formatting. Look like you don't have properly configured go tools?